lib/main_common.pm

```diff
@@ -272,6 +274,10 @@ sub is_kernel_test {
         || get_var('TUNED'));
 }
 
+sub is_systemd_test {
+    return get_var('SYSTEMD_TESTSUITE');
```

Comment:
is this variable documented in the md file? if not lets add it

Reply:
Not yet, after I will get a got from systemd guys I will add it there. Thanks for the reminder.